README.org

```diff
@@ -11,6 +11,7 @@
      3. i'll make the process better at some point
 ** Mongo
    1. honestly i have no idea. I don't think any special setup is required besides installing and having mongo running.
+   2. tested on mongodb version 3.2.9
```

Comment:
I also did some testing on version 3.2.10 and everything seemed fine at the moment.